spec/models/support/requests/brexit_slug_fetcher_spec.rb

```diff
@@ -0,0 +1 @@
+
```

Comment:
Very minimal XD

`BrexitSlugFetcher` is complex enough code that it needs test coverage though.

Reply:
oops, meant to delete this file! will have a think about ways I can test it